src/integration_tests/schemas/e2e_schema_integration_test.py

```diff
@@ -85,3 +85,11 @@ def test_schema_e2e(self):
 
             assert latest_version_schema_response.status_code == 200
             assert latest_version_schema_response.json() == test_schema
+
+            set_guid_schema_response = session.get(
```

Comment:
The double "f" looks quite confusing as it is being passed in as one argument, what do you think about something more along the lines of this `f"{config.API_URL}/v2/schema?guid={schema['guid']}"` ?

Reply:
That's a typo, thanks Mike. Will change